db/migrate/20211203172235_add_timezone_to_developers.rb

```diff
@@ -1,5 +1,5 @@
 class AddTimezoneToDevelopers < ActiveRecord::Migration[7.0]
   def change
-    add_column :developers, :time_zone, :string
+    add_column :developers, :primary_time_zone, :string
```

Comment:
We can't change an existing migration because it has already been run. Instead, we would need to create a new one that renames the column.

However, I think we can change the label in the form instead of the database column.

Reply:
Oh I see, that would have been disastrous! Done a bit of reading up and it's been a good learning experience(thankfully not expensive). Okay let me work on that :)

Reply:
Btw just for clarification 
I was thinking of generating a new migration to change the column name, would you prefer that or to change the label in the form?

Reply:
Let's change the label in the form, for now.

Reply:
It looks like you added a new migration – can you remove that and instead change the label in the form?

This will make sure your database is in a good state.

```bash
rails db:rollback
rm db/migrate/20211203172235_add_timezone_to_developers.rb
```